src/c++/perf_analyzer/main.cc

```diff
@@ -735,6 +738,11 @@ Usage(char** argv, const std::string& msg = std::string())
              "eg:--model-repository=/tmp/host/docker-data/model_unit_test.",
              18)
       << std::endl;
+  std::cerr << FormatMessage(
+                   " --verbose-csv: The csv files generated by PA will include "
```

Comment:
Do any docs need to be updated?

Reply:
Meh, the -f option is only causally mentioned in a section that probably shouldn't talk about the new format.  I don't think we need to make any doc changes.